recipes/pyswmm/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pyswmm" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4814caddeefe38bad421d792a26d6e9bed9945436cdbfaa9847193e0808f98c
+  patches:
+    - __init__.py.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
These should be in host, not build. Also, does this really not link againt the library?

Reply:
pyswmm is a Ctypes interface. The C library is not needed until using the Python module.